terra_notebook_utils/vcf.py

```diff
@@ -88,6 +87,18 @@ def with_file(cls, filepath, read_buf: memoryview=None):
                 raw.seek(0)
                 return cls.with_gzip_fileobj(raw)
 
+def vcf_info(uri: str,
+             google_billing_project: Optional[str]=WORKSPACE_GOOGLE_PROJECT) -> VCFInfo:
+    if uri.startswith("drs://"):
+        client, drs_info = drs.resolve_drs_for_gs_storage(uri)
+        blob = client.bucket(drs_info.bucket_name, user_project=google_billing_project).get_blob(drs_info.key)
+        return VCFInfo.with_blob(blob)
+    if uri.startswith("gs://"):
+        bucket, key = uri[5:].split("/", 1)
+        blob = gs.get_client().bucket(bucket, user_project=google_billing_project).get_blob(key)
+        return VCFInfo.with_blob(blob)
+    else:
+        return VCFInfo.with_file(uri)
```

Comment:
Should this check (and potentially fail with `NotImplementedError`) for non-local schema like `s3://`?

Reply:
Will add